doc/examples.rst

```diff
@@ -282,3 +282,27 @@ For packing deb package call:
 
 Now the result package may be distributed and installed using dpkg command.
 The package will be installed in /usr/share/tarantool/package_name directory.
+
+Working with tt daemon
+----------------------
+
+You can manage TT daemon with following commands:
+
+* ``tt daemon start`` - launch of a daemon
+* ``tt daemon stop`` - terminate of the daemon
+* ``tt daemon status`` - get daemon status
+* ``tt daemon restart`` - daemon restart
+
+.. code-block:: bash
+
+   $ tt daemon start
+   • Starting tt daemon...
+
+To send request to daemon you can use CURL:
```

Comment:
Add a few words of description. What happens when I send this command? What arguments should I get and why such? Add some commands to the example.

Reply:
Work scenario with an example of running a command with flags has been added.